### PR TITLE
Move to pyproject.toml based package

### PR DESCRIPTION
**Pull request by mferrera**
Fixes the same setuptools-scm issues we've had elsewhere, as this is currently breaking the docs build. Requires another PR to fmu-docs to change the build script.

### CODE REVIEW

**Review by oysteoh — APPROVED** [review]

**Comment by mferrera** [comment]
Thanks! I don't have write access here so someone else will need to merge it @oysteoh

**Comment by oysteoh** [comment]
And i can't merge it either - not sure who has ownership here? @berland  maybe?

**Comment by berland** [comment]
Apparently I could!

**Pull request merged**
